### PR TITLE
allow docker set lower_case_table_names for tenant

### PR DESCRIPTION
**Pull request by 3tieto**
allow docker set lower_case_table_names for tenant

### CODE REVIEW

**Comment by LINxiansheng** [comment]
please sign cla

File: tools/docker/standalone/boot/_env [hunk 1]
```diff
@@ -9,4 +9,5 @@ OB_MEMORY_LIMIT=${OB_MEMORY_LIMIT:-6G}
 OB_DATAFILE_SIZE=${OB_DATAFILE_SIZE:-5G}
 OB_LOG_DISK_SIZE=${OB_LOG_DISK_SIZE:-5G}
 OB_ROOT_PASSWORD=${OB_ROOT_PASSWORD:-}
-OB_SYSTEM_MEMORY=${OB_SYSTEM_MEMORY:-1G}
\ No newline at end of file
+OB_SYSTEM_MEMORY=${OB_SYSTEM_MEMORY:-1G}
+OB_TENANT_LOWER_CASE_TABLE_NAMES=${OB_TENANT_LOWER_CASE_TABLE_NAMES:1}
```
Comment (LINxiansheng) [inline tools/docker/standalone/boot/_env:13]: 应当是OB_TENANT_LOWER_CASE_TABLE_NAMES=${OB_TENANT_LOWER_CASE_TABLE_NAMES:-1}

File: tools/docker/standalone/init_store_for_fast_start.tmp.py [hunk 2]
```diff
@@ -116,8 +117,8 @@ def check_file_or_path_exist(bin_abs_path, home_abs_path, store_tar_file_path, e
                             args.tenant_name, args.tenant_name, args.zone))
             logging.info("waiting for create tenant...")
             create_tenant_begin = datetime.datetime.now()
-            cursor.execute("create tenant %s replica_num=1,zone_list=('%s'),primary_zone='RANDOM',resource_pool_list=('%s_pool') set ob_tcp_invited_nodes='%%', ob_compatibility_mode = 'mysql'" % ( \
-                            args.tenant_name, args.zone, args.tenant_name))
+            cursor.execute("create tenant %s replica_num=1,zone_list=('%s'),primary_zone='RANDOM',resource_pool_list=('%s_pool') set ob_tcp_invited_nodes='%%', ob_compatibility_mode = 'mysql', lower_case_table_names=%s" % ( \
```
Comment (LINxiansheng) [inline tools/docker/standalone/init_store_for_fast_start.tmp.py:120]: lower_case_table_names=%s
 这里用%d会不会更好一些

Comment (3tieto) [reply tools/docker/standalone/init_store_for_fast_start.tmp.py]: fxied in new push

**Comment by 3tieto** [comment]
> please sign cla 请签署 CLA

签了，问题都修复

**Review by LINxiansheng — APPROVED** [review]
LGTM

**Pull request merged**